[32mINFO    [0m integration.ha.test_backups:test_backups.py:490 Remove s3-credentials relation
[32mINFO    [0m juju.model:model.py:2958 Waiting for model:
  main/0 [idle] active: 
[32mINFO    [0m juju.model:model.py:2958 Waiting for model:
  main/0 [idle] active: 
[32mINFO    [0m juju.model:model.py:2958 Waiting for model:
  main/0 [idle] active: 
[32mINFO    [0m integration.ha.test_backups:test_backups.py:502 Re-add s3-credentials relation
[32mINFO    [0m juju.model:model.py:2958 Waiting for model:
  main/0 [idle] active: 
[32mINFO    [0m juju.model:model.py:2958 Waiting for model:
  main/0 [executing] maintenance: Backup setup started.
[32mINFO    [0m juju.model:model.py:2958 Waiting for model:
  main/0 [idle] active: 
[32mINFO    [0m juju.model:model.py:2958 Waiting for model:
  main/0 [idle] active: 
[32mINFO    [0m juju.model:model.py:2958 Waiting for model:
  main/0 [idle] active: 
[32mINFO    [0m integration.ha.test_backups:test_backups.py:511 Syncing credentials for microceph
[32mINFO    [0m juju.model:model.py:2958 Waiting for model:
  main/0 [idle] active: 
  s3-integrator/0 [idle] active: 
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.157.72.133:9200/_recovery?human
[32mINFO    [0m integration.ha.helpers:helpers.py:247 Continuous writes result: namespace(max_stored_id=359, count=360, last_expected_id=359)
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.157.72.133:9200/series_index/_search_shards
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: POST -- https://10.157.72.133:9200/series_index/_refresh
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.157.72.133:9200/series_index/_count
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.157.72.133:9200/_recovery?human
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: POST -- https://10.157.72.133:9200/series_index/_refresh
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.157.72.133:9200/series_index/_count
[32mINFO    [0m integration.ha.helpers:helpers.py:591 Testing restore for series_index - original count pre-restore: 360, and now, new count: 323
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: POST -- https://10.157.72.133:9200/series_index/_refresh
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.157.72.133:9200/series_index/_count
[32mINFO    [0m integration.ha.conftest:conftest.py:39 



The writes have been cleared.




[32mINFO    [0m pytest_operator.plugin:plugin.py:862 Model status:

Model  Controller           Cloud/Region         Version  SLA          Timestamp
test   localhost-localhost  localhost/localhost  3.4.4    unsupported  03:05:17Z

App                       Version  Status  Scale  Charm                     Channel        Rev  Exposed  Message
failover                           active      2  opensearch                                 2  no       
main                               active      1  opensearch                                 0  no       
opensearch                         active      1  opensearch                                 1  no       
s3-integrator                      active      1  s3-integrator             latest/edge     31  no       
self-signed-certificates           active      1  self-signed-certificates  latest/stable  155  no       

Unit                         Workload  Agent  Machine  Public address  Ports     Message
failover/0*                  active    idle   3        10.157.72.231   9200/tcp  
failover/1                   active    idle   4        10.157.72.87    9200/tcp  
main/0*                      active    idle   5        10.157.72.133   9200/tcp  
opensearch/0*                active    idle   2        10.157.72.91    9200/tcp  
s3-integrator/0*             active    idle   0        10.157.72.192             
self-signed-certificates/0*  active    idle   1        10.157.72.58              

Machine  State    Address        Inst id        Base          AZ  Message
0        started  10.157.72.192  juju-1bdef7-0  ubuntu@22.04      Running
1        started  10.157.72.58   juju-1bdef7-1  ubuntu@22.04      Running
2        started  10.157.72.91   juju-1bdef7-2  ubuntu@22.04      Running
3        started  10.157.72.231  juju-1bdef7-3  ubuntu@22.04      Running
4        started  10.157.72.87   juju-1bdef7-4  ubuntu@22.04      Running
5        started  10.157.72.133  juju-1bdef7-5  ubuntu@22.04      Running

[32mINFO    [0m pytest_operator.plugin:plugin.py:868 Juju error logs:

machine-0: 02:44:03 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-0: 02:44:03 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-s3-integrator-0: 02:44:03 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-1: 02:44:04 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-1: 02:44:04 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-self-signed-certificates-0: 02:44:04 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-3: 02:44:44 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-3: 02:44:44 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-failover-0: 02:44:44 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-4: 02:44:56 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-4: 02:44:56 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-2: 02:44:56 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-2: 02:44:56 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-5: 02:44:56 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-5: 02:44:56 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-failover-1: 02:44:56 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
unit-main-0: 02:44:56 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
unit-opensearch-0: 02:44:56 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
unit-failover-0: 02:49:32 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-0: 02:49:33 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:49:33 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:49:34 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:49:34 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:49:35 ERROR unit.failover/0.juju-log upgrade-version-a:6: [Errno 111] Connection refused
unit-failover-0: 02:49:36 ERROR unit.failover/0.juju-log upgrade-version-a:6: [Errno 111] Connection refused
unit-failover-0: 02:49:37 ERROR unit.failover/0.juju-log node-lock-fallback:5: [Errno 111] Connection refused
unit-failover-1: 02:49:38 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-opensearch-0: 02:49:38 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-failover-0: 02:49:38 ERROR unit.failover/0.juju-log peer-cluster-orchestrator:12: [Errno 111] Connection refused
unit-main-0: 02:49:39 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-failover-1: 02:49:39 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:49:39 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-opensearch-0: 02:49:39 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:49:39 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-opensearch-0: 02:49:40 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:49:40 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:49:40 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-main-0: 02:49:40 ERROR unit.main/0.juju-log opensearch-peers:8: [Errno 111] Connection refused
unit-main-0: 02:49:41 ERROR unit.main/0.juju-log opensearch-peers:8: [Errno 111] Connection refused
unit-main-0: 02:49:41 ERROR unit.main/0.juju-log opensearch-peers:8: [Errno 111] Connection refused
unit-main-0: 02:49:41 ERROR unit.main/0.juju-log opensearch-peers:8: [Errno 111] Connection refused
unit-failover-1: 02:49:41 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:49:41 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-opensearch-0: 02:49:41 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-main-0: 02:49:42 ERROR unit.main/0.juju-log peer-cluster-orchestrator:10: [Errno 111] Connection refused
unit-failover-1: 02:49:43 ERROR unit.failover/1.juju-log certificates:14: [Errno 111] Connection refused
unit-opensearch-0: 02:49:44 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-main-0: 02:49:44 ERROR unit.main/0.juju-log upgrade-version-a:7: [Errno 111] Connection refused
unit-failover-0: 02:49:45 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 02:49:45 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:49:45 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:49:45 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:49:45 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-main-0: 02:49:46 ERROR unit.main/0.juju-log opensearch-peers:8: [Errno 111] Connection refused
unit-main-0: 02:49:46 ERROR unit.main/0.juju-log opensearch-peers:8: [Errno 111] Connection refused
unit-main-0: 02:49:46 ERROR unit.main/0.juju-log opensearch-peers:8: [Errno 111] Connection refused
unit-main-0: 02:49:46 ERROR unit.main/0.juju-log opensearch-peers:8: [Errno 111] Connection refused
unit-failover-0: 02:49:46 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 02:49:46 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-main-0: 02:49:47 ERROR unit.main/0.juju-log peer-cluster-orchestrator:11: [Errno 111] Connection refused
unit-failover-0: 02:49:47 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-1: 02:49:47 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-opensearch-0: 02:49:47 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-failover-0: 02:49:48 ERROR unit.failover/0.juju-log node-lock-fallback:5: [Errno 111] Connection refused
unit-main-0: 02:49:48 ERROR unit.main/0.juju-log peer-cluster-orchestrator:10: [Errno 111] Connection refused
unit-failover-1: 02:49:48 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-opensearch-0: 02:49:49 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:49:49 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:49:49 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:49:49 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:49:49 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-main-0: 02:49:49 ERROR unit.main/0.juju-log certificates:13: [Errno 111] Connection refused
unit-failover-1: 02:49:49 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-opensearch-0: 02:49:50 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-failover-0: 02:49:50 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:49:50 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:49:50 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:49:50 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:49:50 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-opensearch-0: 02:49:51 ERROR unit.opensearch/0.juju-log certificates:15: [Errno 111] Connection refused
unit-failover-0: 02:49:53 ERROR unit.failover/0.juju-log peer-cluster-orchestrator:12: [Errno 111] Connection refused
unit-failover-1: 02:49:53 ERROR unit.failover/1.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-0: 02:49:54 ERROR unit.failover/0.juju-log peer-cluster-orchestrator:12: [Errno 111] Connection refused
unit-failover-1: 02:49:54 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-main-0: 02:49:56 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-failover-0: 02:49:56 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 02:49:56 ERROR unit.failover/1.juju-log node-lock-fallback:5: [Errno 111] Connection refused
unit-failover-1: 02:49:57 ERROR unit.failover/1.juju-log node-lock-fallback:5: [Errno 111] Connection refused
unit-main-0: 02:49:57 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-failover-0: 02:49:57 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-opensearch-0: 02:49:57 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-failover-1: 02:49:58 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:49:58 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:49:58 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-1: 02:49:58 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:49:58 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:49:58 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:49:58 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:49:58 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-main-0: 02:49:58 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 02:49:59 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-failover-1: 02:49:59 ERROR unit.failover/1.juju-log certificates:14: [Errno 111] Connection refused
unit-main-0: 02:49:59 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-main-0: 02:50:00 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-main-0: 02:50:00 ERROR unit.main/0.juju-log Cannot connect to the OpenSearch server...
unit-main-0: 02:50:00 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 02:50:00 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-failover-0: 02:50:00 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-main-0: 02:50:00 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-main-0: 02:50:00 ERROR unit.main/0.juju-log Cannot connect to the OpenSearch server...
unit-main-0: 02:50:00 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-main-0: 02:50:00 ERROR unit.main/0.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:50:01 ERROR unit.opensearch/0.juju-log upgrade-version-a:3: [Errno 111] Connection refused
unit-failover-1: 02:50:01 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-0: 02:50:01 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-1: 02:50:02 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-0: 02:50:02 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-opensearch-0: 02:50:03 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-main-0: 02:50:04 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-main-0: 02:50:04 ERROR unit.main/0.juju-log Cannot connect to the OpenSearch server...
unit-main-0: 02:50:07 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-main-0: 02:50:08 ERROR unit.main/0.juju-log Cannot connect to the OpenSearch server...
unit-main-0: 02:50:12 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-main-0: 02:50:13 ERROR unit.main/0.juju-log Cannot connect to the OpenSearch server...
unit-main-0: 02:50:16 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-main-0: 02:50:17 ERROR unit.main/0.juju-log Cannot connect to the OpenSearch server...
unit-failover-1: 02:50:17 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-main-0: 02:50:20 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-main-0: 02:50:20 ERROR unit.main/0.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:50:22 ERROR unit.opensearch/0.juju-log certificates:15: [Errno 111] Connection refused
unit-failover-0: 02:50:22 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-1: 02:50:22 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-main-0: 02:50:23 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-main-0: 02:50:23 ERROR unit.main/0.juju-log Cannot connect to the OpenSearch server...
unit-failover-0: 02:50:23 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 02:50:23 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-failover-1: 02:50:24 ERROR unit.failover/1.juju-log peer-cluster-orchestrator:12: [Errno 111] Connection refused
unit-failover-0: 02:50:25 ERROR unit.failover/0.juju-log upgrade-version-a:6: [Errno 111] Connection refused
unit-opensearch-0: 02:50:26 ERROR unit.opensearch/0.juju-log certificates:15: [Errno 111] Connection refused
unit-failover-1: 02:50:26 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:50:26 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:50:26 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-main-0: 02:50:26 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-main-0: 02:50:26 ERROR unit.main/0.juju-log Cannot connect to the OpenSearch server...
unit-failover-0: 02:50:27 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 02:50:27 ERROR unit.failover/1.juju-log upgrade-version-a:6: [Errno 111] Connection refused
unit-opensearch-0: 02:50:28 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-failover-0: 02:50:28 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:50:28 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:50:28 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:50:28 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:50:29 ERROR unit.failover/1.juju-log peer-cluster-orchestrator:12: [Errno 111] Connection refused
unit-opensearch-0: 02:50:29 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-main-0: 02:50:29 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-main-0: 02:50:29 ERROR unit.main/0.juju-log Cannot connect to the OpenSearch server...
unit-failover-0: 02:50:29 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:50:30 ERROR unit.failover/1.juju-log upgrade-version-a:6: [Errno 111] Connection refused
unit-opensearch-0: 02:50:30 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-failover-0: 02:50:30 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:50:30 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:50:31 ERROR unit.failover/1.juju-log peer-cluster-orchestrator:12: [Errno 111] Connection refused
unit-failover-0: 02:50:31 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:50:31 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:50:31 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:50:31 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:50:31 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:50:31 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:50:31 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-opensearch-0: 02:50:31 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-main-0: 02:50:32 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-main-0: 02:50:33 ERROR unit.main/0.juju-log Cannot connect to the OpenSearch server...
unit-failover-1: 02:50:33 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 02:50:34 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:50:34 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:50:34 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:50:34 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:50:34 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-opensearch-0: 02:50:34 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:50:34 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:50:34 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:50:34 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:50:34 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:50:35 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:50:35 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:50:35 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-opensearch-0: 02:50:35 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-main-0: 02:50:36 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-main-0: 02:50:36 ERROR unit.main/0.juju-log Cannot connect to the OpenSearch server...
unit-failover-1: 02:50:36 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-opensearch-0: 02:50:36 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:50:36 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:50:36 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:50:36 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:50:37 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-opensearch-0: 02:50:38 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-failover-1: 02:50:38 ERROR unit.failover/1.juju-log upgrade-version-a:6: [Errno 111] Connection refused
unit-main-0: 02:50:39 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-main-0: 02:50:39 ERROR unit.main/0.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:50:39 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-opensearch-0: 02:50:40 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-failover-0: 02:50:54 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-0: 02:50:54 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-1: 02:51:00 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-opensearch-0: 02:51:00 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-failover-0: 02:51:00 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 02:51:01 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 02:51:01 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 02:51:01 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-opensearch-0: 02:51:01 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:51:01 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:51:01 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:51:01 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-opensearch-0: 02:51:01 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:51:02 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 02:51:02 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 02:51:02 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-opensearch-0: 02:51:02 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-failover-1: 02:51:03 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-0: 02:51:03 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-1: 02:51:03 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-0: 02:51:03 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-1: 02:51:04 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:51:04 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:51:04 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:51:05 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:51:05 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:51:05 ERROR unit.failover/0.juju-log opensearch-peers:4: Cannot connect to the OpenSearch server...
unit-failover-0: 02:51:05 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:51:05 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:51:05 ERROR unit.failover/1.juju-log opensearch-peers:4: Cannot connect to the OpenSearch server...
unit-failover-0: 02:51:05 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:51:05 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:51:05 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:51:05 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:51:05 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:51:05 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:51:05 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:51:06 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:51:06 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:51:06 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:51:06 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:51:06 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:51:06 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:51:06 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:51:06 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:51:06 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:51:06 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:51:06 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:51:06 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:51:06 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:51:07 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:51:07 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:51:07 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:51:07 ERROR unit.failover/1.juju-log opensearch-peers:4: Cannot connect to the OpenSearch server...
unit-failover-1: 02:51:07 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-opensearch-0: 02:51:08 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-failover-0: 02:51:08 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-0: 02:51:08 ERROR unit.failover/0.juju-log Cannot connect to the OpenSearch server...
unit-failover-0: 02:51:08 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-0: 02:51:08 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-0: 02:51:08 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-0: 02:51:08 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-1: 02:51:11 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:51:14 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:51:18 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:51:21 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:51:25 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-opensearch-0: 02:51:26 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:51:28 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-opensearch-0: 02:51:28 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:51:29 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:51:29 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-opensearch-0: 02:51:29 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:51:29 ERROR unit.failover/0.juju-log peer-cluster:10: Cannot connect to the OpenSearch server...
unit-failover-0: 02:51:29 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 02:51:30 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 02:51:30 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 02:51:30 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 02:51:31 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:51:32 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 02:51:32 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 02:51:33 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 02:51:33 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 02:51:33 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 02:51:33 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 02:51:34 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:51:34 ERROR unit.failover/0.juju-log opensearch-peers:4: Cannot connect to the OpenSearch server...
unit-failover-0: 02:51:34 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:51:34 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:51:34 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:51:35 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:51:35 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:51:35 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:51:35 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:51:36 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:51:36 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:51:36 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:51:37 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-opensearch-0: 02:51:38 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-failover-0: 02:51:38 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 02:51:38 ERROR unit.failover/0.juju-log peer-cluster:10: Cannot connect to the OpenSearch server...
unit-failover-0: 02:51:38 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 02:51:39 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 02:51:39 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 02:51:39 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-opensearch-0: 02:51:39 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-failover-0: 02:51:39 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 02:51:40 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 02:51:40 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 02:51:40 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 02:51:40 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:51:41 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 02:51:41 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-opensearch-0: 02:51:42 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-failover-0: 02:51:42 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 02:51:42 ERROR unit.failover/0.juju-log peer-cluster:10: Cannot connect to the OpenSearch server...
unit-failover-0: 02:51:42 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 02:51:42 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 02:51:43 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 02:51:43 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-opensearch-0: 02:51:43 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:51:43 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:51:43 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:51:43 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:51:43 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:51:44 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 02:51:44 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 02:51:45 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 02:51:45 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 02:51:45 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 02:51:45 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 02:51:46 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:51:51 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:51:53 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:51:53 ERROR unit.failover/0.juju-log opensearch-peers:4: Cannot connect to the OpenSearch server...
unit-failover-0: 02:51:53 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:51:53 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:51:53 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:51:54 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:51:54 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:51:54 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:51:54 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:51:54 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:51:54 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:51:55 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:51:55 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:51:55 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:51:55 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:51:56 ERROR unit.failover/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-opensearch-0: 02:51:57 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-failover-0: 02:51:57 ERROR unit.failover/0.juju-log upgrade-version-a:6: [Errno 111] Connection refused
unit-failover-0: 02:51:57 ERROR unit.failover/0.juju-log upgrade-version-a:6: Cannot connect to the OpenSearch server...
unit-failover-0: 02:51:57 ERROR unit.failover/0.juju-log upgrade-version-a:6: [Errno 111] Connection refused
unit-failover-1: 02:51:57 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:51:57 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-1: 02:51:58 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:51:58 ERROR unit.failover/0.juju-log upgrade-version-a:6: [Errno 111] Connection refused
unit-failover-0: 02:51:58 ERROR unit.failover/0.juju-log upgrade-version-a:6: [Errno 111] Connection refused
unit-failover-0: 02:51:58 ERROR unit.failover/0.juju-log upgrade-version-a:6: [Errno 111] Connection refused
unit-failover-0: 02:51:58 ERROR unit.failover/0.juju-log upgrade-version-a:6: [Errno 111] Connection refused
unit-opensearch-0: 02:51:58 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:51:58 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:51:58 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:51:58 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:51:58 ERROR unit.failover/0.juju-log upgrade-version-a:6: [Errno 111] Connection refused
unit-failover-0: 02:52:00 ERROR unit.failover/0.juju-log upgrade-version-a:6: [Errno 111] Connection refused
unit-failover-0: 02:52:00 ERROR unit.failover/0.juju-log upgrade-version-a:6: Cannot connect to the OpenSearch server...
unit-failover-0: 02:52:00 ERROR unit.failover/0.juju-log upgrade-version-a:6: [Errno 111] Connection refused
unit-failover-1: 02:52:00 ERROR unit.failover/1.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-failover-0: 02:52:03 ERROR unit.failover/0.juju-log upgrade-version-a:6: [Errno 111] Connection refused
unit-failover-0: 02:52:07 ERROR unit.failover/0.juju-log upgrade-version-a:6: [Errno 111] Connection refused
unit-failover-0: 02:52:10 ERROR unit.failover/0.juju-log upgrade-version-a:6: [Errno 111] Connection refused
unit-failover-0: 02:52:13 ERROR unit.failover/0.juju-log upgrade-version-a:6: [Errno 111] Connection refused
unit-failover-0: 02:52:17 ERROR unit.failover/0.juju-log upgrade-version-a:6: [Errno 111] Connection refused
unit-failover-0: 02:52:20 ERROR unit.failover/0.juju-log upgrade-version-a:6: [Errno 111] Connection refused
unit-failover-0: 02:52:23 ERROR unit.failover/0.juju-log upgrade-version-a:6: [Errno 111] Connection refused
unit-failover-0: 02:52:26 ERROR unit.failover/0.juju-log upgrade-version-a:6: [Errno 111] Connection refused
unit-failover-0: 02:52:29 ERROR unit.failover/0.juju-log upgrade-version-a:6: [Errno 111] Connection refused
unit-failover-0: 02:52:32 ERROR unit.failover/0.juju-log upgrade-version-a:6: [Errno 111] Connection refused
unit-failover-0: 02:52:35 ERROR unit.failover/0.juju-log upgrade-version-a:6: [Errno 111] Connection refused
unit-failover-0: 02:52:38 ERROR unit.failover/0.juju-log upgrade-version-a:6: [Errno 111] Connection refused
unit-failover-0: 02:52:42 ERROR unit.failover/0.juju-log upgrade-version-a:6: [Errno 111] Connection refused
unit-opensearch-0: 02:52:55 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-opensearch-0: 02:52:56 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-opensearch-0: 02:52:56 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-opensearch-0: 02:52:57 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-opensearch-0: 02:52:58 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 02:53:00 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:53:01 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:53:01 ERROR unit.opensearch/0.juju-log opensearch-peers:1: Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:53:01 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:53:01 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:53:01 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:53:02 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:53:02 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:53:03 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:53:03 ERROR unit.opensearch/0.juju-log opensearch-peers:1: Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:53:04 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:53:07 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:53:11 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:53:14 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:53:19 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:53:22 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:53:26 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:53:29 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:53:32 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:53:35 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:53:38 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:53:41 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:53:44 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:53:47 ERROR unit.opensearch/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-main-0: 02:55:51 ERROR unit.main/0.juju-log s3-credentials:16: Plugin repository-s3 missing: ['access-key', 'secret-key']
unit-main-0: 02:55:54 ERROR unit.main/0.juju-log s3-credentials:16: Plugin repository-s3 missing: ['access-key', 'secret-key']
unit-main-0: 02:55:57 ERROR unit.main/0.juju-log s3-credentials:16: Plugin repository-s3 missing: ['access-key', 'secret-key']
unit-main-0: 02:56:09 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.157.72.133:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 5)
	Error: 500 Server Error: Internal Server Error for url: https://10.157.72.133:9200/_snapshot/s3-repository
unit-main-0: 02:56:10 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.157.72.133:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 4)
	Error: 500 Server Error: Internal Server Error for url: https://10.157.72.133:9200/_snapshot/s3-repository
unit-main-0: 02:56:12 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.157.72.133:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 3)
	Error: 500 Server Error: Internal Server Error for url: https://10.157.72.133:9200/_snapshot/s3-repository
unit-main-0: 02:56:13 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.157.72.133:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 2)
	Error: 500 Server Error: Internal Server Error for url: https://10.157.72.133:9200/_snapshot/s3-repository
unit-main-0: 02:56:15 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.157.72.133:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 1)
	Error: 500 Server Error: Internal Server Error for url: https://10.157.72.133:9200/_snapshot/s3-repository
unit-main-0: 02:56:16 ERROR unit.main/0.juju-log s3-credentials:16: Failed to setup backup service with state repository s3 is unreachable
unit-s3-integrator-0: 02:56:26 ERROR juju.worker.dependency "uniter" manifold worker returned unexpected error: relation scope id "b135a318-ef4c-406a-8943-3d7cc21bdef7:r#17#failover": settings b135a318-ef4c-406a-8943-3d7cc21bdef7:r#17#failover not found (not found)
unit-opensearch-0: 02:56:27 ERROR juju.worker.dependency "uniter" manifold worker returned unexpected error: relation scope id "b135a318-ef4c-406a-8943-3d7cc21bdef7:r#18#s3-integrator": settings b135a318-ef4c-406a-8943-3d7cc21bdef7:r#18#s3-integrator not found (not found)
unit-main-0: 02:56:31 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.157.72.133:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 5)
	Error: 500 Server Error: Internal Server Error for url: https://10.157.72.133:9200/_snapshot/s3-repository
unit-main-0: 02:56:32 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.157.72.133:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 4)
	Error: 500 Server Error: Internal Server Error for url: https://10.157.72.133:9200/_snapshot/s3-repository
unit-main-0: 02:56:34 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.157.72.133:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 3)
	Error: 500 Server Error: Internal Server Error for url: https://10.157.72.133:9200/_snapshot/s3-repository
unit-main-0: 02:56:35 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.157.72.133:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 2)
	Error: 500 Server Error: Internal Server Error for url: https://10.157.72.133:9200/_snapshot/s3-repository
unit-main-0: 02:56:36 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.157.72.133:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 1)
	Error: 500 Server Error: Internal Server Error for url: https://10.157.72.133:9200/_snapshot/s3-repository
unit-main-0: 02:56:38 ERROR unit.main/0.juju-log s3-credentials:16: Failed to setup backup service with state repository s3 is unreachable
unit-main-0: 02:56:57 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.157.72.133:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.106.187.3', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/43cd9564-9c2a-4e6e-9b8a-6a99a43ab3c0', 'region': 'default'}} failed.(Attempts left: 5)
	Error: 500 Server Error: Internal Server Error for url: https://10.157.72.133:9200/_snapshot/s3-repository
unit-main-0: 02:56:59 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.157.72.133:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.106.187.3', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/43cd9564-9c2a-4e6e-9b8a-6a99a43ab3c0', 'region': 'default'}} failed.(Attempts left: 4)
	Error: 500 Server Error: Internal Server Error for url: https://10.157.72.133:9200/_snapshot/s3-repository
unit-main-0: 02:57:00 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.157.72.133:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.106.187.3', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/43cd9564-9c2a-4e6e-9b8a-6a99a43ab3c0', 'region': 'default'}} failed.(Attempts left: 3)
	Error: 500 Server Error: Internal Server Error for url: https://10.157.72.133:9200/_snapshot/s3-repository
unit-main-0: 02:57:02 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.157.72.133:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.106.187.3', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/43cd9564-9c2a-4e6e-9b8a-6a99a43ab3c0', 'region': 'default'}} failed.(Attempts left: 2)
	Error: 500 Server Error: Internal Server Error for url: https://10.157.72.133:9200/_snapshot/s3-repository
unit-main-0: 02:57:03 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.157.72.133:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.106.187.3', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/43cd9564-9c2a-4e6e-9b8a-6a99a43ab3c0', 'region': 'default'}} failed.(Attempts left: 1)
	Error: 500 Server Error: Internal Server Error for url: https://10.157.72.133:9200/_snapshot/s3-repository
unit-failover-0: 02:57:15 ERROR unit.failover/0.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.access_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.access_key] does not exist in the keystore.

 Stdout: 
unit-opensearch-0: 02:57:15 ERROR unit.opensearch/0.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.access_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.access_key] does not exist in the keystore.

 Stdout: 
unit-failover-0: 02:57:17 ERROR unit.failover/0.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.access_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.access_key] does not exist in the keystore.

 Stdout: 
unit-opensearch-0: 02:57:18 ERROR unit.opensearch/0.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.access_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.access_key] does not exist in the keystore.

 Stdout: 
unit-failover-0: 02:57:19 ERROR unit.failover/0.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.access_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.access_key] does not exist in the keystore.

 Stdout: 
unit-opensearch-0: 02:57:20 ERROR unit.opensearch/0.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.access_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.access_key] does not exist in the keystore.

 Stdout: 
unit-failover-0: 02:57:21 ERROR unit.failover/0.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.secret_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.secret_key] does not exist in the keystore.

 Stdout: 
unit-opensearch-0: 02:57:22 ERROR unit.opensearch/0.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.secret_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.secret_key] does not exist in the keystore.

 Stdout: 
unit-failover-0: 02:57:23 ERROR unit.failover/0.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.secret_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.secret_key] does not exist in the keystore.

 Stdout: 
unit-opensearch-0: 02:57:24 ERROR unit.opensearch/0.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.secret_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.secret_key] does not exist in the keystore.

 Stdout: 
unit-main-0: 02:57:25 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.157.72.133:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.106.187.3', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/43cd9564-9c2a-4e6e-9b8a-6a99a43ab3c0', 'region': 'default'}} failed.(Attempts left: 5)
	Error: 500 Server Error: Internal Server Error for url: https://10.157.72.133:9200/_snapshot/s3-repository
unit-failover-0: 02:57:25 ERROR unit.failover/0.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.secret_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.secret_key] does not exist in the keystore.

 Stdout: 
unit-opensearch-0: 02:57:26 ERROR unit.opensearch/0.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.secret_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.secret_key] does not exist in the keystore.

 Stdout: 
unit-main-0: 02:57:26 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.157.72.133:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.106.187.3', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/43cd9564-9c2a-4e6e-9b8a-6a99a43ab3c0', 'region': 'default'}} failed.(Attempts left: 4)
	Error: 500 Server Error: Internal Server Error for url: https://10.157.72.133:9200/_snapshot/s3-repository
unit-main-0: 02:57:28 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.157.72.133:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.106.187.3', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/43cd9564-9c2a-4e6e-9b8a-6a99a43ab3c0', 'region': 'default'}} failed.(Attempts left: 3)
	Error: 500 Server Error: Internal Server Error for url: https://10.157.72.133:9200/_snapshot/s3-repository
unit-main-0: 02:57:29 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.157.72.133:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.106.187.3', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/43cd9564-9c2a-4e6e-9b8a-6a99a43ab3c0', 'region': 'default'}} failed.(Attempts left: 2)
	Error: 500 Server Error: Internal Server Error for url: https://10.157.72.133:9200/_snapshot/s3-repository
unit-main-0: 02:57:31 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.157.72.133:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.106.187.3', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/43cd9564-9c2a-4e6e-9b8a-6a99a43ab3c0', 'region': 'default'}} failed.(Attempts left: 1)
	Error: 500 Server Error: Internal Server Error for url: https://10.157.72.133:9200/_snapshot/s3-repository
unit-main-0: 03:01:23 ERROR unit.main/0.juju-log s3-credentials:19: Request PUT to https://10.157.72.133:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.106.187.3', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/43cd9564-9c2a-4e6e-9b8a-6a99a43ab3c0', 'region': 'default'}} failed.(Attempts left: 5)
	Error: 500 Server Error: Internal Server Error for url: https://10.157.72.133:9200/_snapshot/s3-repository
unit-main-0: 03:01:25 ERROR unit.main/0.juju-log s3-credentials:19: Request PUT to https://10.157.72.133:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.106.187.3', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/43cd9564-9c2a-4e6e-9b8a-6a99a43ab3c0', 'region': 'default'}} failed.(Attempts left: 4)
	Error: 500 Server Error: Internal Server Error for url: https://10.157.72.133:9200/_snapshot/s3-repository
unit-main-0: 03:01:26 ERROR unit.main/0.juju-log s3-credentials:19: Request PUT to https://10.157.72.133:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.106.187.3', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/43cd9564-9c2a-4e6e-9b8a-6a99a43ab3c0', 'region': 'default'}} failed.(Attempts left: 3)
	Error: 500 Server Error: Internal Server Error for url: https://10.157.72.133:9200/_snapshot/s3-repository
unit-main-0: 03:01:28 ERROR unit.main/0.juju-log s3-credentials:19: Request PUT to https://10.157.72.133:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.106.187.3', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/43cd9564-9c2a-4e6e-9b8a-6a99a43ab3c0', 'region': 'default'}} failed.(Attempts left: 2)
	Error: 500 Server Error: Internal Server Error for url: https://10.157.72.133:9200/_snapshot/s3-repository
unit-main-0: 03:01:29 ERROR unit.main/0.juju-log s3-credentials:19: Request PUT to https://10.157.72.133:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.106.187.3', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/43cd9564-9c2a-4e6e-9b8a-6a99a43ab3c0', 'region': 'default'}} failed.(Attempts left: 1)
	Error: 500 Server Error: Internal Server Error for url: https://10.157.72.133:9200/_snapshot/s3-repository
unit-main-0: 03:01:47 ERROR unit.main/0.juju-log s3-credentials:19: Request PUT to https://10.157.72.133:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.106.187.3', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/43cd9564-9c2a-4e6e-9b8a-6a99a43ab3c0', 'region': 'default'}} failed.(Attempts left: 5)
	Error: 500 Server Error: Internal Server Error for url: https://10.157.72.133:9200/_snapshot/s3-repository
unit-main-0: 03:01:48 ERROR unit.main/0.juju-log s3-credentials:19: Request PUT to https://10.157.72.133:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.106.187.3', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/43cd9564-9c2a-4e6e-9b8a-6a99a43ab3c0', 'region': 'default'}} failed.(Attempts left: 4)
	Error: 500 Server Error: Internal Server Error for url: https://10.157.72.133:9200/_snapshot/s3-repository
unit-main-0: 03:01:50 ERROR unit.main/0.juju-log s3-credentials:19: Request PUT to https://10.157.72.133:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.106.187.3', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/43cd9564-9c2a-4e6e-9b8a-6a99a43ab3c0', 'region': 'default'}} failed.(Attempts left: 3)
	Error: 500 Server Error: Internal Server Error for url: https://10.157.72.133:9200/_snapshot/s3-repository
unit-main-0: 03:01:51 ERROR unit.main/0.juju-log s3-credentials:19: Request PUT to https://10.157.72.133:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.106.187.3', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/43cd9564-9c2a-4e6e-9b8a-6a99a43ab3c0', 'region': 'default'}} failed.(Attempts left: 2)
	Error: 500 Server Error: Internal Server Error for url: https://10.157.72.133:9200/_snapshot/s3-repository
unit-main-0: 03:01:53 ERROR unit.main/0.juju-log s3-credentials:19: Request PUT to https://10.157.72.133:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.106.187.3', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/43cd9564-9c2a-4e6e-9b8a-6a99a43ab3c0', 'region': 'default'}} failed.(Attempts left: 1)
	Error: 500 Server Error: Internal Server Error for url: https://10.157.72.133:9200/_snapshot/s3-repository

[32mINFO    [0m pytest_operator.plugin:plugin.py:947 Forgetting model main...
[32mINFO    [0m integration.ha.test_backups:test_backups.py:173 Cleaning backups from cloud buckets